client/web/web.go

```diff
@@ -58,6 +58,7 @@ type Server struct {
 	devMode    bool
 	cgiMode    bool
 	pathPrefix string
+	readonly   bool
```

Comment:
not used?

Reply:
good catch.  That was from before I decided to add a new server mode.